src/geo.c

```diff
@@ -554,9 +558,11 @@ void georadiusGeneric(client *c, int srcKeyIndex, int flags) {
             } else if (!strcasecmp(arg, "count") && (i+1) < remaining) {
                 if (getLongLongFromObjectOrReply(c, c->argv[base_args+i+1],
                     &count, NULL) != C_OK) return;
-                if (count <= 0) {
-                    addReplyError(c,"COUNT must be > 0");
+                if (count == 0) {
+                    addReplyError(c,"COUNT must be greater than 0 or less than 0");
                     return;
+                } else if (count < 0) {
+                    limit = -count;
```

Comment:
any reason why we don't wanna normalize `count` here (setting it to 0), instead of the `llabs` you added below?

Reply:
good suggestion, already updated.